oshc/main/views.py

```diff
@@ -38,3 +38,9 @@ def add_contest(request):
 
 def submit_contest(request):
     return render(request, 'contest_submission.html')
+
+
+def journey(request):
+    journey_list = Journey.objects.all()
+    return render(request, 'journey.html',
+                  context={'Journey': journey_list})
```

Comment:
You might want to sort the `journey_list` by the `start_date` before sending it to the HTML page. This ensures that the data we send out of the view is chronologically ordered.
You can use `Managers`[0] to achieve this or you could simply do something like `Journey.objects.all().filteryBy('start_date')` 

[0] - https://docs.djangoproject.com/en/2.0/topics/db/managers/

Reply:
fixed w/ `Journey.objects.order_by('start_date')`.